code/game/gamemodes/revolution/revolution.dm

```diff
@@ -554,4 +541,70 @@
 	dat += "<b>Revolution Successful:</b> [score_traitorswon ? "Yes" : "No"] (-[score_traitorswon * 10000] Points)<br>"
 	dat += "<HR>"
 
-	return dat
\ No newline at end of file
+	return dat
+
+
+
+/obj/effect/proc_holder/spell/targeted/recruitrev
+	name = "Recruit Revolutionary"
+	desc = "Allows you to recruit a conscious, non-braindead, non-catatonic non-loyal human to be part of the revolution."
+	charge_max = 450
+	clothes_req = 0
+	range = 1 //Adjacent to user
+	action_icon_state = "spell_greytide"
+	var/recruiting = 0
+
+/obj/effect/proc_holder/spell/targeted/recruit/cast(list/targets)
+	for(var/mob/living/carbon/human/target in targets)
+		if(!in_range(usr, target))
+			usr << "<span class='warning'>You need to be closer to recruit [target].</span>"
+			charge_counter = charge_max
+			return
+		if(!target.ckey)
+			usr << "<span class='warning'>The target has no mind.</span>"
+			charge_counter = charge_max
+			return
+		if(target.stat)
+			usr << "<span class='warning'>The target must be conscious.</span>"
+			charge_counter = charge_max
+			return
+		if(!ishuman(target))
+			usr << "<span class='warning'>You can only recruit humans.</span>"
+			charge_counter = charge_max
+			return
+		if(recruiting)
+			usr << "<span class='danger'>You are already recruiting!</span>"
+			charge_counter = charge_max
+			return
+		recruiting = 1
+		usr << "<span class='danger'>This target is valid. You begin the recruiting process.</span>"
+		target << "<span class='userdanger'>[usr] focuses in concentration. Your head begins to ache.</span>"
+
+		for(var/progress = 0, progress <= 3, progress++)
+			switch(progress)
+				if(1)
+					usr << "<span class='notice'>You begin by introducing yourself and explaining what you're about.</span>"
+					usr.visible_message("<span class='danger'>[usr]'s introduces himself and explains his plans.</span>")
+				if(2)
+					usr << "<span class='notice'>You begin the recruitment of [target].</span>"
+					usr.visible_message("<span class='danger'>[usr] leans over towards [target], whispering excitedly as he gives a speech.</span>")
+					target << "<span class='danger'>You feel yourself agreeing with [usr], and a surge of loyalty begins building.</span>"
+					sleep(20)
+					if(isloyal(target))
+						usr << "<span class='notice'>They are enslaved by Nanotrasen. You feel their interest in your cause wane and disappear.</span>"
+						usr.visible_message("<span class='danger'>[usr] stops talking for a moment, then moves back away from [target].</span>")
+						target << "<span class='danger'>Your loyalty implant activates and a sharp pain reminds you of your loyalties to Nanotrasen.</span>"

```

Comment:
The implants are geared towards "preventing mind control and similar effects" rather then forcing loyalty. So this shall be a tricky fluff to come up with if it's meant to be consensual.